packages/code-editor/CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## [0.2.0](https://www.github.com/looker-open-source/sdk-codegen/compare/code-editor-v0.1.1...code-editor-v0.2.0) (2021-05-11)
+
+
+### Features
+
+* reduce size of apix bundle ([#676](https://www.github.com/looker-open-source/sdk-codegen/issues/676)) ([0d74f6a](https://www.github.com/looker-open-source/sdk-codegen/commit/0d74f6a7814b509416a9d2558c16439a4859b543))
```

Comment:
oops, code-editor missing `bump-minor-pre-major` and `bump-patch-for-minor-pre-major` - this should be `0.1.2` - fix: #679